### PR TITLE
Extended-DatedVehicleJourney+Replace-PR368

### PR DESCRIPTION
**Pull request by Aurige**
Initial PR368 from Kristian Syversen (Kirstian has moved and the PR was not editable ... so it was easier to create a new one). 

This is the exact copy of PR368, but targeting 'next' and not master branch 
It complements it with deprecated DriverRef (DatedJourney are expected to be referenced by driver's DUTY, not the way arround). 

This is conforming the decision from meeting #3 of the NeTEx revision 2022/2023.

### CODE REVIEW

**Review by JohanEntur — APPROVED** [review]

**Review by ue71603 — APPROVED** [review]

**Comment by Mike-Stallybrass** [comment]
This pull request does not yet appear to fulfil the functional needs.  The original need was related to the use of DatedServiceJourneys, not just to the use of DatedVehicleJourneys.  
There remains a need to be able to refer to a **_ServiceJourney_** (or more generally a _**Journey),**_ from within a **_DatedServiceJourney_**.   And the new **_ReplacedJourneys_RelStructure_**  needs to be available within a **_DatedServiceJourney_** 
However, because there continues to be a distinction between a **_DatedServiceJourneyGroup_** and a **_DatedVehicleJourneyGroup_**, and the changes are only being made to a **_DatedVehicleJourneysGroup_**, the functional needs that triggered this pull request will **NOT** be met by committing this pull request

**Comment by Aurige** [comment]
Isn't it ok with the way XSD Substitution Groups are configured ?

**Comment by Mike-Stallybrass** [comment]
Hi Christophe,

In applying either PR #368 to Master, or #411 to Next, I found that the result did not actually alter the situation for a DatedServiceJourney, only for a DatedVehicleJourney, yet it is in the context of a DatedServiceJourney that all the trace-back of previous service journey opportunities is really necessary.
Also, I have reuploaded the CR using CR204, as I found the master sheet, where the highest number in use was 203.

Regards,

Mike Stallybrass  |  Principal Consultant
Signature Rail  |  www.signaturerail.com <http://www.signaturerail.com/>  |  ***@***.******@***.***>
Jervaulx House, 6 St. Mary’s Court, Blossom Street, York, YO24 1AH, UK
t: +44 (0)1904 639091  | m: +44 (0) 7540 824941
***@***.***

Registered in England number 02484158. Registered Office: as above.

**Comment by hfjelstad** [comment]
I approve the latest changes

**Comment by Aurige** [comment]
> I approve the latest changes

can you do a "formal" approval/review ? we need 2 before being able to merge

**Comment by skinkie** [comment]
@Aurige he can't because he does not have write access to the repo

File: xsd/netex_part_2/part2_journeyTimes/netex_datedVehicleJourney_version.xsd
Comment (skinkie) [inline xsd/netex_part_2/part2_journeyTimes/netex_datedVehicleJourney_version.xsd:312]:
```suggestion
					<xsd:attribute name="id" type="NormalDatedVehicleJourneyIdType" use="required">
```

**Comment by skinkie** [comment]
In my opinion you can merge.

**Comment by skinkie** [comment]
@Aurige it is reverted now.

**Pull request merged**
